partcad/src/partcad/part_factory_alias.py

```
@@ -63,29 +67,36 @@ def __init__(self, ctx, source_project, target_project, config):
 
             self.ctx.stats_parts_instantiated += 1
 
-            if source.path:
-                part.path = source.path
-            return await source.instantiate(part)
+            return await source.instantiate(obj)
 
     def get_final_config(self):
         source = self.ctx._get_part(self.source)
         if not source:
             raise Exception(f"The alias source {self.source} is not found")
```

Ruff (0.8.2)

Create your own exception

(TRY002)